q2_taxa/plugin_setup.py

```diff
@@ -59,6 +76,10 @@
     function=tabulate,
     inputs={'data': FeatureData[Taxonomy]},
     parameters={},
+    input_descriptions={
+        'data': 'Taxonomic annotations for all features.',
```

Comment:
"all features" doesn't make sense here because there isn't a corresponding feature table. What about *Taxonomic annotations associated with features.* ? Or: *Features with associated taxonomic annotations.*